### DISCUSSION TITLE
Navigation buttons

### DISCUSSION THREAD

**Discussion by rasimcakir** [alexkaessner/macOS-UI-Kit#11] [Ideas & Suggestions]
Hello Alex
First of all I would like to thank you for your excellent work. Looks just great! Could you please integrate the navigation buttons (see screenshot) in the status normal, hover and pressed? Thank you very much.

Take care,
Rasim

<img width="875" alt="Screenshot 2021-01-15 at 16 37 33" src="https://user-images.githubusercontent.com/76400135/104747908-5f6eda80-5751-11eb-8601-65045f90e70d.png">

**Comment by alexkaessner** [comment]
Hi @rasimcakir, sorry for the super late reply. I couldn’t find time to work on an update. It’s a great suggestion though and will look into this with the next improvements. Thanks for that hint!

**Comment by alexkaessner** [comment]
@rasimcakir Check out the newest update v2.1! Besides new alert symbols it also includes new symbols like you’ve suggested:
<img width="835" alt="CleanShot 2021-03-07 at 18 47 59@2x" src="https://user-images.githubusercontent.com/6615076/110249265-afaa3200-7f75-11eb-838a-8657f63a5b44.png">
